frontend/workflows/envoy/src/remote-triage/index.tsx

```diff
@@ -122,6 +130,9 @@ const TriageDetails: React.FC<WizardChild> = () => {
 };
 
 const RemoteTriage: React.FC<WorkflowProps> = ({ heading }) => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const hostParam = urlParams.get("q");
```

Comment:
We use q for the resolver right now so we shouldn't overwrite it. Since this is scoped to only the remote triage workflow we should prefix it with `_`